### DISCUSSION TITLE
directus_files bigInteger vs integer

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15474] [Archive]
![image](https://user-images.githubusercontent.com/8686739/189077187-7683b7bb-0bab-4012-83db-e2af0c1c782c.png)

**Comment by rijkvanzanten** [comment]
You're comparing the up migration with the down migration. The `down()` call undoes the migration, so it goes back to the previous type. This migration changes the column from `integer` to `bigInteger`

**Reply by anlexN** [reply]
why go back to the previous type?

**Reply by rijkvanzanten** [reply]
You need to be able to downgrade any upgrade you've done. If for whatever reason you need to go back to a previous version, the down migration is used to roll back
